docs/changelogs/v0.21.md

```diff
@@ -6,13 +6,24 @@
 
 - [Overview](#overview)
 - [🔦 Highlights](#-highlights)
+  - [Accelerated DHT Client is no longer experimental](#--empty-repo-is-now-the-default)
 - [📝 Changelog](#-changelog)
 - [👨‍👩‍👧‍👦 Contributors](#-contributors)
 
 ### Overview
 
 ### 🔦 Highlights
 
+#### Accelerated DHT Client is no longer experimental
+
+The accelerated DHT client is now the main recomended solution for peoples who
+are hosting lots of data.
+
+We moved the `Experimental.AcceleratedDHTClient` flag in `Routing.AcceleratedDHTClient`.
+
+A new tracker will try to estimate the providing speed and warn users that they
+should be using AcceleratedDHTClient because they are falling behind.
```

Comment:
```suggestion
A new tracker estimates the providing speed and warns users if they
should be using AcceleratedDHTClient because they are falling behind.
```